lib/voyager_helpers/liberator.rb

```diff
@@ -572,11 +569,10 @@ def exec_get_info_for_item(query, item_id, conn, full)
           info[:enum] = valid_ascii(enum)
           chron = row.shift
           info[:chron] = valid_ascii(chron)
-          date = row.shift
-          info[:status_date] = date.to_datetime unless date.nil?
           info[:barcode] = row.shift
         end
-        if ['Charged', 'Renewed', 'Overdue'].include? info[:status]
+        info[:status] = get_item_statuses(item_id, conn)
+        if (info[:status].keys & ['Charged', 'Renewed', 'Overdue'])
```

Comment:
Oh, I think you need to update this: `unless (info[:status].keys & ['Charged', 'Renewed', 'Overdue']).empty?`